apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/RuntimeTestsApi.ts

```diff
@@ -209,5 +185,5 @@ export async function recordAnimationUpdates() {
 }
 
 export async function stopRecordingAnimationUpdates() {
-  testRunner.stopRecordingAnimationUpdates();
+  await testRunner.stopRecordingAnimationUpdates();
```

Comment:
Was it caught by eslint? Is it how it was intended to work?

Reply:
Yes, there should be `await` here, and preciously eslint didn't catch it.